plugin/trino-kafka/src/main/java/io/trino/plugin/kafka/KafkaClientsModule.java

```diff
@@ -30,6 +31,7 @@
     protected void setup(Binder binder)
     {
         configBinder(binder).bindConfig(KafkaSecurityConfig.class);
+        installClientModule(null, KafkaClientsModule::configureDefault);
```

Comment:
Instead of not specifying any modules, can we delegate to `DefaultText...` for other SecurityProtocol ?

Reply:
Hi @Praveen2112 , thanks for reviewing this PR. Can you kindly provide an example of what you mean by 

> Instead of not specifying any modules, can we delegate to DefaultText... for other SecurityProtocol`

This line currently does install the default module.